migrations/versions/0308_delete_loadtesting_provider.py

```diff
@@ -0,0 +1,28 @@
+"""
+Remove loadtesting provider
+
+Revision ID: 0308_delete_loadtesting_provider
+Revises: 0307_delete_dm_datetime
+Create Date: 2019-10-22 17:30
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0308_delete_loadtesting_provider'
+down_revision = '0307_delete_dm_datetime'
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute("DELETE FROM provider_details WHERE identifier = 'loadtesting'")
+    conn.execute("DELETE FROM provider_details_history WHERE identifier = 'loadtesting'")
+
+def downgrade():
+    conn = op.get_bind()
+    conn.execute(
+        """
+        INSERT INTO provider_details (id, display_name, identifier, priority, notification_type, active, updated_at, version, created_by_id, supports_international) VALUES
```

Comment:
Is this line valid SQL? No defined values on the end.

Feels unlikely that we would downgrade but just checking

Reply:
no ill fix that

Reply:
i have fixed that